src/Server.js

```diff
@@ -85,11 +85,15 @@ class Server {
     const Config = await IoC.use("Config");
 
     this.app.get("/", (req, res) => {
-      res.json({
+      const defaultAxeResponse = {
```

Comment:
I think that we can put `defaultAxeResponse` to the `src/constants.js` file.

In addition, all `/docs/*` routes are visible in the only the development environment. [Check it out](https://github.com/axe-api/axe-api/blob/master/src/Server.js#L95). So, for now we don't need to add these routes to the default API response.